docs/docusaurus/src/css/navbar.scss

```diff
@@ -25,6 +25,13 @@
   box-shadow: unset;
   height: unset;
   padding: 0;
+
+  &:has(.home-section.navbar__link--active),
+  &:has(.gx-cloud-section.navbar__link--active) {
+    .navbar__items .dropdown {
+      display: none;
```

Comment:
I know the CSS solution is not the ideal, but this is mainly coming from the need we have to fix this ASAP and we're really limited by the tools that docusaurus allows us to use in terms o versioning or the components, as we're trying to avoid the need of ejecting the dropdown version component which is the ideal place to do the changes but it can bring a lot of problems when updating to newer versions of docusaurus. Maybe we can bring @joshzzheng to discussion here but yesterday we talked and doing this logic in CSS seemed like the best option in this moment